terraform/lib/dependabot/terraform/file_fetcher.rb

```diff
@@ -47,6 +51,33 @@ def terragrunt_files
           map { |f| fetch_file_from_host(f.name) }
       end
 
+      def local_path_module_files(files, dir: ".")
+        terraform_files = []
+
+        files.each do |file|
+          terraform_file_local_module_details(file).each do |path|
+            base_path = Pathname.new(File.join(dir, path)).cleanpath.to_path
+            nested_terraform_files =
+              repo_contents(dir: base_path).
+              select { |f| f.type == "file" && f.name.end_with?(".tf") }.
+              map { |f| fetch_file_from_host(File.join(base_path, f.name)) }
+            terraform_files += nested_terraform_files
+            terraform_files += local_path_module_files(nested_terraform_files, dir: path)
+          end
+        end
+
+        terraform_files.tap { |fs| fs.each { |f| f.support_file = true } }
```

Comment:
Are `support_files` handled differently?
I don't see how this value is used.

Reply:
Yeah we don't use it differently for terraform but we tag path dependency files this way for other ecosystems so I was only thinking of maintaining that data model. Will add a comment explaining this.